docs/rules/expect.md

```diff
@@ -0,0 +1,141 @@
+# expect-type/expect
+
+💼 This rule is enabled in the ✅ `recommended` config.
+
+🔧 This rule is automatically fixable by the [`--fix` CLI option](https://eslint.org/docs/latest/user-guide/command-line-interface#--fix).
+
+💭 This rule requires type information.
+
+<!-- end auto-generated rule header -->
+
+Enforces that types indicated in special comments match the types of code values.
+
+## Comment Types
+
+The following kinds of comments are supported:
+
+<!-- Markdownlint doesn't seem to understand the heading IDs... -->
+<!-- markdownlint-disable link-fragments -->
+
+- [`^?`](#twoslash)
+- [`$ExpectType`](#expecttype)
+- [`$ExpectTypeSnapshot`](#expecttypesnapshot)
+- [`$ExpectError`](#expecterror)
+
+<!-- markdownlint-enable link-fragments -->
+
+### Twoslash Syntax (`^?`)
+
+[Twoslash](https://www.typescriptlang.org/dev/twoslash) is the term for special comment directives in TypeScript code for tools such as this one.
+This plugin supports "twoslash assertions", which are comments containing `^?` and a type below a value in code to check that the value is that type.
+
+For example:
+
+```ts
+declare const getTextLength: (text: string) => number;
+
+getTextLength('abc');
+// ^? number
+
+getTextLength;
+// ^? (text: string) => number
+```
+
+Mismatching the type will cause a lint report:
+
+```ts
+'abc';
+// ^? number
+```
+
+```plaintext
+Expected type to be: number, got: string
+```
+
+Multiline type annotations are also supported:
+
+```ts
+const vector = {
+  x: 3,
+  y: 4,
+};
+vector;
+// ^? const vector: {
+//        x: number;
+//        y: number;
+//    }
+```
+
+### `$ExpectError`
+
+Asserts that a particular line of code.
```

Comment:
It looks like something got cut off here.

Reply:
Whoops!